teleport/tests/test_common.py

```diff
@@ -11,11 +11,13 @@
 pytestmark = [pytest.mark.unit]
 
 
-def test_connect_exception(dd_run_check):
+def test_connect_exception(dd_run_check, caplog):
     with pytest.raises(Exception):
```

Comment:
Should we check some info about the Exception being raised? What happens if we get an Exception by a different reason?

Reply:
You can assert the exception message with the match parameter ie:
`with pytest.raises(Exception, match="error message"):`

Reply:
Done!